server/src/main/java/io/deephaven/server/barrage/BarrageStreamGenerator.java

```diff
@@ -768,10 +918,10 @@ private ByteBuffer getSubscriptionMetadata(final SubView view) throws IOExceptio
         final int nodesOffset = metadata.endVector();
 
         BarrageUpdateMetadata.startBarrageUpdateMetadata(metadata);
-        BarrageUpdateMetadata.addNumAddBatches(metadata,
-                LongSizedDataStructure.intSize("BarrageStreamGenerator", view.numAddBatches));
-        BarrageUpdateMetadata.addNumModBatches(metadata,
-                LongSizedDataStructure.intSize("BarrageStreamGenerator", view.numModBatches));
+        BarrageUpdateMetadata.addNumAddBatches(metadata, LongSizedDataStructure.intSize("BarrageStreamGenerator",
```

Comment:
The concept of number of batches is deprecated IMO.  We should change this to number of rows or remove (since num rows is implicit in rowsIncluded / rowsModified).

Reply:
submitted barrage PR (https://github.com/deephaven/barrage/pull/28)